src/winsvc.c

```diff
@@ -296,7 +296,7 @@ static int lua_StartService(lua_State *L) {
   return 2;
 }
 
-static void svcdispatcher_end_cb(uv_handle_t* handle) {
+static void svcdispatcher_end_cb(uv_async_t* handle) {
```

Comment:
This for avoid warning when compling,
According to uv_sync_cb ptototype 
int uv_async_init(uv_loop_t*,
                            uv_async_t* async,
                            uv_async_cb async_cb);